src/Turbo/src/Helper/TurboStream.php

```diff
@@ -86,12 +86,34 @@ public static function refresh(?string $requestId = null): string
         return \sprintf('<turbo-stream action="refresh" request-id="%s"></turbo-stream>', htmlspecialchars($requestId));
     }
 
-    private static function wrap(string $action, string $target, string $html, string $attr = ''): string
+    /**
+     * Custom action and attributes.
+     *
+     * @param array<string, string|int|null> $attr
+     */
+    public static function custom(string $action, string $target, string $html, array $attr = []): string
```

Comment:
I'd rather keep the wrap method, and have this new method use it 

(prepare and escape the $attr string and pass it to self::wrap())